aries_vcx/src/protocols/connection/mod.rs

```diff
@@ -100,7 +100,13 @@ where
         message: &AriesMessage,
     ) -> VcxResult<EncryptionEnvelope> {
         let sender_verkey = &self.pairwise_info().pw_vk;
-        EncryptionEnvelope::create(wallet, message, Some(sender_verkey), self.their_did_doc()).await
+        EncryptionEnvelope::create(
+            wallet,
+            json!(message).to_string().as_bytes(),
```

Comment:
does the `json!` macro have potential for panic here?

according to the docs, it will panic if the `Serialize` impl fails.

i wonder if it'd just be better to use `serde_json::to_vec?` instead of this macro wish might panic.

Reply:
same as below

Reply:
I personally think this can be disregarded, *for now*. This is not some problem introduced by this PR, but was there for a long time. This whole encryption thing needs a rework, IMO.

Reply:
Since the code was initially passing in `AriesMessage` struct (for which Serialize is implemented locally) I think the possibility of panics in these instances is remote. 

Still, I could add the `serde_json::to_vec?`, if that's better, no problem.

Reply:
Oh sorry, I see now that it was already doing this potentially panicky behaviour. No need to fix right now